code/modules/goals/station_goals/bluespace_tap.dm

```diff
@@ -405,7 +419,7 @@
 	points -= A.product_cost
 	playsound(src, 'sound/magic/blink.ogg', 50)
 	do_sparks(2, FALSE, src)
-	new A.product_path(get_turf(src))
+	new A.product_path(get_step(src,(dir)))
```

Comment:
same here

Reply:
Spawning on top machine is bad so i moved it to the south where there's a tray to contain

Reply:
Looks better that way